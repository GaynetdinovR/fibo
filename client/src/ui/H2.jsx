import styles from "../styles/Ui.module.sass";
import classNames from "classnames";

const H2 = ({ children, className }) => {
	return (
		<h2 className={classNames(className, styles.h2)}>
			{children}
		</h2>
	);
};

export default H2;