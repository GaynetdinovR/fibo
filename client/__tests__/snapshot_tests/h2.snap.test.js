import renderer from "react-test-renderer";
import { it, expect } from "@jest/globals";
import H2 from "../../src/ui/H2.jsx";

it("renders correctly", () => {
	const tree = renderer.create(<H2>H2</H2>).toJSON();
	expect(tree).toMatchSnapshot();
});
