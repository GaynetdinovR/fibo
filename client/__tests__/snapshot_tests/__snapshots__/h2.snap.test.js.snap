// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<h2
  className="h2"
>
  H2
</h2>
`;
